src/Symfony/Component/Mailer/Bridge/Amazon/Transport/SesTransportFactory.php

```diff
@@ -31,6 +32,10 @@ public function create(Dsn $dsn): TransportInterface
         $port = $dsn->getPort();
 
         if ('ses+api' === $scheme) {
+            if (!class_exists(\SimpleXMLElement::class)) {
```

Comment:
Can't we replace this with an `\extension_loaded()` check? that'd allow opcache to optimize the code path.

Reply:
Great idea 👍

Reply:
Done